frontstage/models.py

```diff
@@ -80,17 +76,44 @@ def validate_password(form, field):
 
 class LoginForm(FlaskForm):
     username = StringField('Email Address', [InputRequired('Email Address is required'),
-                                             Email("Your email should be of the form myname@email.com ")])
+                                             Email('Invalid email address')])
     password = PasswordField('Password', [InputRequired('Password is required')])
 
+    @staticmethod
+    def validate_username(form, field):
+        return _validate_email_address(field)
+
+
+def _validate_email_address(field):
```

Comment:
Now that these validation checks have been moved into a separate function, perhaps we should have unit tests for it?

Reply:
Was just looking in to this, will add tests accordingly